views/maintenance.hbs

```diff
@@ -8,7 +8,7 @@
   <title>{{title}}</title>
   <link rel="stylesheet" href="/public/css-reset/bundle.css" media="all" />
   <link rel="stylesheet" href="/public/atlassian-ui-kit/bundle.css" media="all" />
-  <link rel="stylesheet" href="/public/css/styles.css" media="all" />
+  <link rel="stylesheet" href="/public/css/global.css" media="all" />
```

Comment:
was this meant to be?

Reply:
Yep. As part of the UI changes I renamed styles as a file with globally shared styling, it makes more sense for the file to be named 'global'.

Reply:
agreed :)